client/src/App.tsx

```diff
@@ -0,0 +1,16 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom'
+import Auth from './pages/Auth';
+import Dashboard from './pages/Dashboard';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path='/login' element={<Auth />} />
```

Comment:
i created a constants folder, and created a `route.ts` file, where I stored the following constants:

`const VERSION = '/v1';
const MEDIA = '/media';
const LOGIN = '/login';
const SIGNUP = '/signup';
const TYPES = '/types';
const ITEMS = '/items';
const ADD_ITEM = '/addItem';
const ID_PARAM = '/:id';`

what do you think about this?

Reply:
yup, should be fine 👍  that's what we usually do also. if you'd need another category of constants stored somewhere, don't hesitate to create another file

Reply:
For these, use enums lang @maxellmilay. The route constants you defined will be much more cohesive that way since may indicator na sila when we do `Route.login` rather than `login`

Reply:
ooh okok, interesting. I'll search that up